packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.js

```diff
@@ -33,31 +33,31 @@ export function getConvoIdFromUrl(convoUrl) {
  * @returns {Boolean|String}
  */
 export function validateAndDecodeId(string) {
-  let resultId = false;
+  if (!(string && string.length)) {
+    return {};
+  }
 
   if (isUuid(string)) {
-    resultId = string;
+    return {
+      id: string
+    };
   }
-  else {
-    let isEncodedUUID = false;
-    let base64uuid;
 
-    // Check for base 64 encoding
-    try {
-      base64uuid = atob(string).split('/').pop();
-      isEncodedUUID = isUuid(base64uuid);
-    }
-    catch (e) {
-      if (e.name !== 'InvalidCharacterError') {
-        throw e;
-      }
+  // Check for base 64 encoding
+  try {
+    const idParts = deconstructHydraId(string);
+
+    if (isUuid(idParts.id)) {
+      return idParts;
     }
-    if (isEncodedUUID) {
-      resultId = base64uuid;
+  }
+  catch (e) {
+    if (e.name !== 'InvalidCharacterError') {
+      throw e;
     }
   }
 
-  return resultId;
+  return {};
```

Comment:
This is no longer returning a false when not verified. Does this affect anything using it downstream? 

Also, could you update the jsdoc to match what the new return value is?

Reply:
The rest of this PR are the changes necessary to move from 'false' to an empty object.